lane.py

```diff
@@ -1157,7 +1158,6 @@ def search(self, query, search_client, pagination=None):
                         lazyload(mw.license_pool, LicensePool.identifier),
                         lazyload(mw.license_pool, LicensePool.presentation_edition),
                     )
-                    q = self.only_show_ready_deliverable_works(q, mw)
```

Comment:
I think it would be valuable to apply at least the filter against `LicensePool.suppressed`. When someone suppresses a license pool it's a sign of a serious problem and they want it to be taken off the shelf _immediately_, not at 3 AM the next morning.